meta/src/main/proto/gravitino_meta.proto

```diff
@@ -61,10 +61,40 @@ message Schema {
   uint64 id = 1;
   string name = 2;
   AuditInfo audit_info = 3;
+  // These two fields will be used when the schema object is managed by Gravitino, otherwise, it
+  // will be ignored.
+  optional string comment = 4;
+  map<string, string> properties = 5;
 }
 
 message Table {
   uint64 id = 1;
   string name = 2;
   AuditInfo audit_info = 3;
-}
\ No newline at end of file
+}
+
+/**
+ * The File message is used to record the file object information. File object represents
+ * the non-tabular data on FS-like storage, like HDFS, S3, etc.
+ */
+message File {
+  enum Format {
+    CSV = 0;
```

Comment:
`UNKNOWN = 0` may be better.  You can see https://blog.51cto.com/u_16162321/6496255